pkg/cli/admin/release/image_mapper.go

```diff
@@ -166,25 +166,30 @@ func NewTransformFromImageStreamFile(path string, input *imageapi.ImageStream, a
 		if !ok {
 			continue
 		}
-		klog.V(4).Infof("Found build versions from %s: %s", tag.Name, value)
-		items, err := parseComponentVersionsLabel(value)
+		displayNameValue, ok := tag.Annotations[annotationBuildVersionsDisplayNames]
+		if !ok {
+			continue
+		}
```

Comment:
will this skip our current images that don't have display names configured yet, only the versions ?

Reply:
Yikes, good catch

Reply:
Going to unit test this pretty thoroughly

Reply:
Sent a fix